main/src/cgeo/geocaching/settings/TokenPreference.java

```diff
@@ -54,6 +54,7 @@ public TokenPreference(final Context context, final AttributeSet attrs, final in
 
     @Override
     protected OnPreferenceClickListener getOnPreferenceClickListener(final SettingsActivity activity) {
+        //todo
         //activity.setAuthTitle(tokenMapping.prefKeyId);
```

Comment:
R.string.settings_reauthorize / R.string.settings_authorize switch

More occurrences of `setAuthTitle` (see below) generalize?